app/src/androidTest/java/com/realityexpander/tasky/EventDaoTest.kt

```diff
@@ -67,7 +67,7 @@ open class EventDaoTest {
             photos = listOf(
                 PhotoEntity("1", "https://www.google.com")
             ),
-            deletedPhotoKeys = emptyList(),
+            deletedPhotoIds = emptyList(),
```

Comment:
In practice, it totally makes sense to put such renamings and refactorings that affect a lot of classes into a separate PR (which doesn't need review for this app), since these things quickly consume half of a PR and you gotta spot the important changes in between

Reply:
OK, I will move this kind of activity to a separate PR